app/src/main/java/org/oppia/android/app/onboarding/IntroFragment.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.app.onboarding
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.FragmentComponentImpl
+import org.oppia.android.app.fragment.InjectableFragment
+import org.oppia.android.util.extensions.getStringFromBundle
+import javax.inject.Inject
+
+/** Fragment that contains the introduction message for new learners. */
+class IntroFragment : InjectableFragment() {
+  @Inject
+  lateinit var introFragmentPresenter: IntroFragmentPresenter
+
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+    (fragmentComponent as FragmentComponentImpl).inject(this)
+  }
+
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
+    val profileNickname = arguments!!.getStringFromBundle(PROFILE_NICKNAME_ARGUMENT_KEY)!!
```

Comment:
We should use ``?.`` to access the bundle, and then a ``checkNotNull()`` as we do elsewhere.

Or, use protos for better type safety & field presence assurance.

Reply:
Added the checkNotNull() option.